projects/backend/capi/__tests__/articleImgPicker.spec.ts

```diff
@@ -78,7 +80,7 @@ describe('articleImgPicker.getImages', () => {
             elements: [thumbnailElem],
         }
 
-        const actual = getImages(given)
+        const actual = getImages(given, articleTypePicker(given))
```

Comment:
Would recommend defining `articleTypePicker(given)` further up just once as this seems to be used as a constant in testing

Reply:
👍